R/ESA.R

```diff
@@ -0,0 +1,85 @@
+# ESA.R
+#
+# Purpose: Exploratory System Analysis tool project for BCB420 2019
+# Version: 1.0.0
+# Version history: 1.0.0
+# Date: 25 March 2019
+# Author: Deus Bajaj (deus.bajaj@mail.utoronto.ca)
+# License: MIT
+#
+# Input: A biological system and a gene the user picks from the system
+# Output: A histogram for the node degree distribution of the system gene network
+# and a network plot for a gene from the biological system
+# Dependencies: devtools, iGraph
```

Comment:
A: move this information into your roxygen header.
B: why is there a `devtools` dependency?